client/landing/gutenboarding/components/header/style.scss

```diff
@@ -23,11 +23,18 @@
 	}
 }
 
-.gutenboarding__header-section {
+.gutenboarding__header-section,
+.gutenboarding__header-back-button {
 	display: flex;
 	align-items: center;
 }
 
+.gutenboarding__header-back-button {
```

Comment:
```suggestion
.gutenboarding__header-back-button {
	color: $light-gray-900;

	&:focus:not(:disabled) {
		color: $dark-gray-300;
	}
	
```

Reply:
TBH that makes the Back button so faint when it's disabled that it's barely visible, and faint enough when it's enabled for me to think that it's disabled :sweat_smile: 

Let's maybe also discuss in a follow-up? I'd be inclined to stick with the defaults for enabled and disabled for now.

(I realize it looks faint in the mockups, but I was inclined to think that it was supposed to be disabled there, and was simply not shown as enabled during a few steps when it should be because it wasn't the main concern of those mockups.)